code/game/area/space_station_13_areas.dm

```diff
@@ -168,7 +168,7 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 /area/maintenance
 	name = "Generic Maintenance"
```

Comment:
Are there any other areas with maint in their name that could use the flag?

Reply:
to my surprise, this was /ACTUALLY/ done consistently. anywhere that should be maintinence is a subtype of this area; the ONE exception is the AI sat, which I'm gonna wait on a specific 'do that' before flagging - it's such an edgecase that I think the whole AI sat would probably just have a unique flag at that point anyways

the one flaw i realized with this is that Radstorm Shelters are ALSO subtypes of maintinence, so they'll get catwalked if they lose tiles. Not a big flaw, but if radstorms ever come back in full force it might be worth a flag specifically for rad-proofing instead of just using maintinence for it

Reply:
Make sure you add the flag to any subtypes that override area_flags, xenobio maint comes to mind.

I feel a bit silly proposing this, but if we just want all the subtypes of /area/maintenance, why not just do `istype(area, /area/maintenance)`? or is having more granular control useful you think

Reply:
well
tbh
I didnt know ALL maints were properly subtypes of maints. Figured someone'd messed that up at some point, so I searched the area's name for "maint";; but surprisingly they havent messed it up so
this bitflag stuff might be a bit pointless thinking about it, but idk

Reply:
you wanna uh, just use the typepath?
Seems close enough to "is this maintenance" to me, genuinely kinda feel bad I didn't realize this earlier

Reply:
yeah i think I'm gonna bail on the bitflag and just check if its area/maintenance

if we need a bitflag, it should be for something more specific, like 'radproof', not just 'maints'; maints really SHOULD just be 'subtype of maints'